packages/wrangler/src/config/environment.ts

```diff
@@ -523,6 +523,8 @@ interface EnvironmentNonInheritable {
 		binding: string;
 		/** The id of the database. */
 		id: string;
+		/** The local postgres connection string for `wrangler dev` */
```

Comment:
Does this only support postgres?

Reply:
Currently Hyperdrive only supports postgres, I can change it do `database connection string` for future safety too though